app/src/main/res/values/strings.xml

```diff
@@ -356,11 +356,11 @@
     <string name="and_i_m_up_for">And I\'m up for</string>
     <string name="what_type">What Sounds good?</string>
     <string name="feedback">Feedback</string>
-    <string name="write_feedback">Write Feedback</string>
+    <string name="write_feedback">WRITE A FEEDBACK</string>
```

Comment:
Revert

Reply:
Okay. I just follow the play store for UI.

Reply:
Button text is automatically capitalized

Reply:
Yes right sorry. I have changed it for text view.

Reply:
Not capitalizing with that style. Added an extra attribute.